default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs ? import <nixpkgs>{} } :
+let
+    buildGoModule = pkgs.buildGoModule;
```

Comment:
Can we move these inline as well?

Reply:
I could convert this part of the code to something like this: 
```
let
    inherit(pkgs)
        buildGoModule
        statik
        git
        runCommand;
in
```

Would that work better?

Reply:
Yeah either works, I suppose it's mainly stylistic haha. Up to you